packages/yoastseo/src/values/Paper.js

```diff
@@ -37,8 +36,6 @@ const defaultAttributes = {
  */
 function Paper( text, attributes ) {
 	this._text = text || "";
-	// Unify whitespaces and non-breaking spaces.
-	this._text = unifyNonBreakingSpace( this._text );
```

Comment:
Good point, will do that!

Reply:
Decided not to remove `unifyAllSpaces` from `sanitizeString.js` because 
1. It basically imports the whole `unifyWhitespace` file, which includes a lot of functions, not just removing `&nbsp;`
2. When I tried to remove the `unifyNonBreakingSpace` function from the `unifyWhitespace` this lead to various errors popping up in the console. So the function seems necessary there.